client/src/components/Question.js

```diff
@@ -1,28 +1,44 @@
 import React from 'react';
-import { Form, Jumbotron } from 'react-bootstrap';
+import { Form, Card } from 'react-bootstrap';
 
 const Question = ({ question, handleAnswerSelect, selectedAnswer = [] }) => {
+  const addClassName = question => {
```

Comment:
```suggestion
  const addClassName = question => (question.qType === 'checkbox') ? 'styled-input-square' : 'styled-input-rounded';
```
If you wanted to, you could change the if-else statement into a ternary expression which then turns this into one single line